lighthouse-plugin-publisher-ads/audits/blocking-load-events.js

```diff
@@ -0,0 +1,230 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const i18n = require('lighthouse/lighthouse-core/lib/i18n/i18n');
+const NetworkRecords = require('lighthouse/lighthouse-core/computed/network-records');
+// @ts-ignore
+const TraceOfTab = require('lighthouse/lighthouse-core/computed/trace-of-tab');
+const {Audit} = require('lighthouse');
+const {getAdCriticalGraph} = require('../utils/graph');
+const {getTimingsByRecord} = require('../utils/network-timing');
+
+/** @typedef {LH.Artifacts.NetworkRequest} NetworkRequest */
+/** @typedef {LH.Gatherer.Simulation.NodeTiming} NodeTiming */
+
+const UIStrings = {
+  title: 'Ads not blocked by load events',
+  failureTitle: 'Avoid waiting on load events',
+  description: 'Waiting on load events increases ad latency. ' +
+    'To speed up ads, eliminate the following load event handlers.',
+  displayValue: '{timeInMs, number, seconds} s blocked',
+  columnEvent: 'Event Name',
+  columnTime: 'Event Time',
+  columnScript: 'Script',
+  columnBlockedUrl: 'Blocked URL',
+  columnFunctionName: 'Function',
```

Comment:
Add to lighthouse-plugin-publisher-ads/messages/en-US.json ?

Reply:
That's no longer necessary after recent i18n changes. There is a script that pulls module.exports UIStrings into a separate file